.goreleaser.yml

```diff
@@ -50,3 +50,21 @@ brews:
         type: optional
     test: |
       system "#{bin}/flux --version"
+publishers:
+  - name: aur-pkg-bin
+    ids:
+      - nix
+    cmd: |
+      eval $(ssh-agent -s)
+      ssh-add <(echo "{{ .Env.AUR_BOT_SSH_PRIVATE_KEY }}")
+      ci/aur/flux-bin/publish.sh {{ .Version }}
```

Comment:
Given all our CI is in/on GitHub, I would maybe just place this in the `.github` directory (`.github/release/aur`?).

Reply:
Makes sense, I was also thinking about where to better place this but `.github` was not what I thought would be the right place. I suppose it should be fine judging by your argument regarding all CI stuff living there.